lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -238,33 +242,47 @@ def self.append_phc_version_if_necessary(append_phc_version_if_next_version_is_n
         end
       end
 
+      # rubocop:disable Metrics/PerceivedComplexity
       private_class_method def self.build_changelog_sections(changelog_sections)
         changelog_sections.reject { |_, v| v.empty? }.map do |section_name, prs|
-          next unless prs.size > 0
+          next if prs.empty?
 
           case section_name
           when :breaking_changes
-            title = "### Breaking Changes"
+            title = "### 💥 Breaking Changes"
           when :fixes
-            title = "### Bugfixes"
+            title = "### 🐞 Bugfixes"
           when :new_features
-            title = "### New Features"
+            title = "### ✨ New Features"
           when :paywalls
-            title = "### RevenueCatUI"
+            title = "### 🖼️ RevenueCatUI"
           when :performance
-            title = "### Performance Improvements"
+            title = "### ⚡ Performance Improvements"
           when :dependency_updates
-            title = "### Dependency Updates"
+            title = "### 📦 Dependency Updates"
+          when :work_in_progress
+            title = "### 🚧 Work in Progress"
+            wip_sections = prs.map do |subsection, items|
+              capitalized_subsection = subsection.split.map(&:capitalize).join(' ')
+              "#### #{capitalized_subsection}\n#{items.join("\n")}"
+            end
           else
-            title = "### Other Changes"
+            title = "### 🔄 Other Changes"
           end
-          "#{title}\n#{prs.join("\n")}"
+          if section_name == :work_in_progress
+            lines = wip_sections
+          else
+            lines = prs
+          end
+          "#{title}\n#{lines.join("\n")}"
         end.join("\n")
       end
 
-      # rubocop:disable Metrics/PerceivedComplexity
       private_class_method def self.get_section_depending_on_types_of_change(change_types)
-        if change_types.include?("pr:breaking")
+        wip_label = change_types.find { |type| type.end_with?(":wip") }
```

Comment:
main reason for it to being a suffix :wip  is that I think we can add another prefix feat: different than pr: for bigger arbitrary features if we want, but that can come in a different future PR since that requires a bigger refactor